### PR TITLE
implement example for connectors

### PR DESCRIPTION
**Pull request by aaryaattrey**

### CODE REVIEW

File: dozer-types/src/ingestion_types.rs [hunk 1]
```diff
@@ -331,13 +341,29 @@ impl S3Storage {
...
+    fn example() -> Self {
+        let s3_details = S3Details {
+            access_key_id: "<your_access_key_id>".to_owned(),
+            secret_access_key: "<your_secret_access_key>".to_owned(),
+            region: "<your_region>".to_owned(),
+            bucket_name: "<your_bucket_name>".to_owned(),
+        };
+        Self {
+            details: s3_details,
+            tables: vec![],
```
Comment (v3g42) [inline dozer-types/src/ingestion_types.rs:354]: Include one table as an example

File: dozer-types/src/ingestion_types.rs [hunk 2]
```diff
@@ -331,13 +341,29 @@ impl S3Storage {
         )
     }
 }
+impl SchemaExample for S3Storage {
+    fn example() -> Self {
+        let s3_details = S3Details {
+            access_key_id: "<your_access_key_id>".to_owned(),
```
Comment (v3g42) [inline dozer-types/src/ingestion_types.rs:347]: leave as an empty string instead. Easier to double click and replace.

File: dozer-types/src/ingestion_types.rs [hunk 3]
```diff
@@ -414,3 +446,105 @@ pub fn default_buffer_size() -> u32 {
 pub fn default_snowflake_poll_interval() -> Duration {
     Duration::from_secs(60)
 }
+
+impl SchemaExample for MongodbConfig {
+    fn example() -> Self {
+        Self {
+            connection_string: "mongodb://<username>:<password>@localhost:27017/<database_name>"
```
Comment (v3g42) [inline dozer-types/src/ingestion_types.rs:453]: use the local defaults that are typically used in a local mongo.

Comment (v3g42) [reply dozer-types/src/ingestion_types.rs:453]: Also include a db_name as a sample.

**Review by v3g42 — CHANGES_REQUESTED** [review]
@aaryaattrey As these are examples,I would rather put example values which closely resemble what is to be expected. Look at postgres as an example.

File: dozer-types/src/ingestion_types.rs [hunk 4]
```diff
@@ -414,3 +453,104 @@ pub fn default_buffer_size() -> u32 {
...
+            adapter: Some("arrow".to_owned()),
+        }
+    }
+}
+
+impl SchemaExample for KafkaConfig {
+    fn example() -> Self {
+        Self {
+            broker: "{{broker_url}}".to_owned(),
+            schema_registry_url: Some("{{schema_registry_url}}".to_owned()),
```
Comment (v3g42) [inline dozer-types/src/ingestion_types.rs:489]: These two are still remaining.

File: dozer-types/src/ingestion_types.rs [hunk 5]
```diff
@@ -414,3 +453,104 @@ pub fn default_buffer_size() -> u32 {
...
+        }
+    }
+}
+
+impl SchemaExample for DeltaLakeConfig {
+    fn example() -> Self {
+        Self {
+            tables: vec![DeltaTable {
+                path: "{{delta_table_path}}".to_owned(),
```
Comment (v3g42) [inline dozer-types/src/ingestion_types.rs:498]: Here as well.

File: dozer-types/src/ingestion_types.rs (hunk 4 above)
Comment (aaryaattrey) [reply dozer-types/src/ingestion_types.rs:489]: Should I put an empty string here, or some default value?

Comment (v3g42) [reply dozer-types/src/ingestion_types.rs:489]: just leave an empty value if you are unsure.

**Review by v3g42 — APPROVED** [review]

**Pull request merged**